package.json

```diff
@@ -13,13 +13,14 @@
     "packages/*"
   ],
   "scripts": {
+    "coverage": "codecov",
     "bootstrap": "lerna bootstrap --no-ci",
     "build": "lerna run build",
     "postinstall": "npm run bootstrap",
     "lint": "tslint --project tsconfig.json",
     "lint:fix": "tslint --project tsconfig.json --fix",
-    "test": "jest",
-    "test:watch": "jest --watchAll"
+    "test": "CI=true jest --coverage",
```

Comment:
some coloring library got updated and CI builds vs normal builds have the bash color in the snapshot: `[1m[32mDevelopment:[39m[22m` vs `Development:`

For now run everything as CI